config/locales/en.yml

```diff
@@ -1187,39 +1187,39 @@ en:
           volcano: "Volcano"
           water: "Water"
           wetland: "Wetland"
-          wood: "Wood"
+          wood: "Woods"
           "yes": "Natural Feature"
         office:
-          accountant: "Accountant"
-          administrative: "Administration"
+          accountant: "Accountant Office"
+          administrative: "Administrative Office"
           advertising_agency: "Advertising Agency"
-          architect: "Architect"
-          association: "Association"
-          company: "Company"
+          architect: "Architect Office"
+          association: "Nonprofit Organization Office"
```

Comment:
Elsewhere you linked to a document that explicitly mentioned associations as examples for "not-for-profit", rather than "nonprofit", organisations. So the qualifier "nonprofit" seems too limited for me here.

Reply:
Good point, “nonprofit” would be a better match for the less common `office=charity` tag. This should probably be changed in id-tagging-schema too. The problem with “association” unqualified is that it usually implies something more like `club=*` or `amenity=social_centre` in this dialect, but it’s rather difficult to give any name to these presets without making regional legal assumptions.

How about “Association Office”? At least the “office” would emphasize that this isn’t likely to be a community meeting hall of some sort.

Reply:
Maybe `Organization Office`?